app/views/landing-pages/league/components/Leaderboard.vue

```diff
@@ -26,10 +26,21 @@ export default {
     title: {
       type: String,
       default: ''
+    },
+    anonymousPlayerName: {
```

Comment:
normally it's good to have name like shouldXyz or isXyz for booleans but not blocking this merge